docs/DeveloperGuide.md

```diff
@@ -80,6 +81,17 @@ We use [Cypress](https://www.cypress.io/) for automated end-to-end front-end tes
 1.  At project root, run `gradlew frontendTest`
 > If you encountered an invalid browser error, please ensure that you have `Chrome` installed in the default installation directory. Otherwise, follow the instructions [here](https://docs.cypress.io/guides/guides/debugging.html#Launching-browsers) to create symbolic links so Cypress can locate `Chrome` in your system.
 
+### (Optional) Using Vue.js devtools for frontend debugging
+1. View and download [Vue.js devtools](https://chrome.google.com/webstore/detail/vuejs-devtools/nhdogjmejiglipccpnnnanhbledajbpd) extention on Chrome web store.
+1. Go the detail page of this extension in Chrome's extension management panel and tick `Allow access to file URLs`. If you are unable to locate it, paste and go to the link: `chrome://extensions/?id=nhdogjmejiglipccpnnnanhbledajbpd` in the address bar of chrome.
```

Comment:
~tick~ `select`
~paste and go to~ **copy** the link ... and visit it on your Chrome.